msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -104,7 +110,7 @@
 			}
 			if (_updatePersistence && this.Persistence)
 			{
-				this.printForParser();
+				this.saveToPersistentData();
```

Comment:
This isn't super great with regards to if a user changes multiple settings at once, and then saves, I'm not sure what the *best* solution for that is, but wanted to be sure that's mentioned